### DISCUSSION TITLE
No feedback when doing a search

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3473] [Accessibility]
Priority level 2 (out of 3) from UW Accessible Technologies

WCAG mapping: 4.1.3 Status Messages

STEPS TO REPRODUCE:
1. Go to the global search and search for something such as "test"
2. Note the announcements upon doing the search.

OBSERVED BEHAVIOR:
There are currently no announcements upon successful (search with results) and unsuccessful (search with no results) searches in the global search. Upon doing the search, there is no indication that changes have occurred on the page or the status of the search results.

EXPECTED BEHAVIOR:
When users do searches and the content of the page dynamically changes, users should be alerted of these changes. In this case, it would be appropriate to announce the number of search results from the given search

**Comment by tsmyre** [comment]
**CIC Response**
Clarification needed: we are updating the results in a region with role="status". Is it recommended to use `role="alert"` or to add `aria-live="assertive"`?

**Washington Reply**
`role="status"` is the recommended attribute to use for search result messages such as this, so we are unsure as to why nothing is being announced.
